src/main/resources/org/csanchez/jenkins/plugins/kubernetes/volumes/DynamicPVCVolume/help-requestsSize.html

```diff
@@ -0,0 +1,2 @@
+Claims, like pods, can request specific quantities of a resource. In this case, the request is for storage. The same resource model applies to both volumes and claims.
+
```

Comment:
If I understand correctly getRequestsSizeOrDefault, if this field is empty then default is 10Gi, should be explained to end users. And it gives an example also.
```suggestion
If this field is empty, the default value of 10Gi will be taken.
```